components/camel-salesforce/camel-salesforce-component/src/main/java/org/apache/camel/component/salesforce/SalesforceConstants.java

```diff
@@ -59,6 +58,12 @@ public final class SalesforceConstants {
     @Metadata(label = "consumer", description = "The client id.", javaType = "String")
     public static final String HEADER_SALESFORCE_CLIENT_ID = "CamelSalesforceClientId";
 
+    // Pub/Sub API headers
+    @Metadata(label = "consumer", description = "The Pub/Sub API replayId.", javaType = "Object")
```

Comment:
Oh looks like these are String types, so can you update javaType = "String" for both of these